Sources/Build/BuildDescription/ProductBuildDescription.swift

```diff
@@ -390,7 +390,13 @@ public final class ProductBuildDescription: SPMBuildCore.ProductBuildDescription
         // Other linker flags.
         for target in self.staticTargets {
             let scope = self.buildParameters.createScope(for: target)
-            flags += scope.evaluate(.OTHER_LDFLAGS)
+            let additionalFlags = scope.evaluate(.OTHER_LDFLAGS)
+            // FIXME: this should be based on the target package's tools version
```

Comment:
IIUC our current convention was to base all behavior on root package's tools version? @neonichu do I understand that correctly?

Reply:
It depends. Mostly we try to apply the specific package's tools-version, but there are some behaviors that are more far reaching where we look at root packages.

I think for more build related things like this that should be localized to the package itself, looking at the specific package is the most appropriate.